client/packages/system/src/Item/Components/ItemVariantSelector/ItemVariantInput.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {
+  EditIcon,
+  TextInputButton,
+  ButtonProps,
+} from '@openmsupply-client/common';
+import { ItemVariantFragment, useItemVariants } from '../../api';
+import { ItemVariantSelector } from './ItemVariantSelector';
+
+interface ItemVariantInputProps {
+  itemId: string;
+  selectedId: string | null;
+  onChange: (itemVariant: ItemVariantFragment | null) => void;
+  displayDoseColumns: boolean;
+  width?: number | string;
+  disabled?: boolean;
+}
+
+export const ItemVariantInput = ({
```

Comment:
separated `input` and `cell` - so input can be used from forms, and cell in the context of tables